docker/prometheus.yml

```diff
@@ -6,7 +6,7 @@ global:
     cluster_name: 'promgen'
 
 rule_files:
-- "/etc/prometheus/promgen.rule.yml"
+- "/etc/prometheus/promgen.rule"
```

Comment:
Ah, this is also slightly tricky because the extension depends on if you're running Prometheus 1.x or 2.x. I think docker-compose should probably default to running 2.x by default, but perhaps I'll put a comment here